packages/sit-onyx/src/components/OnyxDialog/OnyxDialog.vue

```diff
@@ -42,6 +42,22 @@ watch([dialog, () => props.open], () => {
   else dialog.value?.close();
 });
 
+watch(dialog, (currentDialog) => {
+  if (props.modal && !props.disableClosingOnBackdropClick) {
+    currentDialog?.addEventListener("click", (e) => {
```

Comment:
Please use a `@click` handler on the dialog directly instead of manually adding it here :)